StorageTests/TestSQLiteHistory.swift

```diff
@@ -1292,7 +1342,8 @@ class TestSQLiteHistory: XCTestCase {
 
             return history.getTopSitesWithLimit(20) >>== { topSites in
                 XCTAssertEqual(topSites.count, 20)
-                XCTAssertEqual(topSites[0]!.guid, "abc\(0)def")
+                XCTAssertEqual(topSites[0]!.guid, "abc\(5)def")
+                XCTAssertEqual(topSites[1]!.guid, "abc\(0)def")
```

Comment:
I think this would be really rare in the real world. And if a site is tied I dont think showing them in a different order would be "incorrect"